test/lit/passes/asyncify_optimize-level=1.wast

```diff
@@ -301,7 +301,11 @@
     (drop (call $import2))
   )
   ;; CHECK:      (func $calls-nothing
-  ;; CHECK-NEXT:  (nop)
+  ;; CHECK-NEXT:  (drop
+  ;; CHECK-NEXT:   (i32.eqz
+  ;; CHECK-NEXT:    (i32.const 17)
+  ;; CHECK-NEXT:   )
+  ;; CHECK-NEXT:  )
   ;; CHECK-NEXT: )
```

Comment:
This is the kind of diff I'd expect: the function after asyncify is identical to before asyncify. We did not modify it at all, showing the optimizer never ran, which is good.